docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3.7'
+
+services:
+  db:
+    image: mongo:4.2
+    volumes:
+      - $PWD/docker/volumes/db:/data/db
+      - $PWD/docker/scripts:/docker-entrypoint-initdb.d
+    networks:
+      - development_network
+    env_file: .env
+
+  app:
```

Comment:
I'm getting the following error. Can you handle it?

```ERROR: for twitch_app_1  Cannot create container for service app: create .: volume name is too short, names should be at least two alphanumeric characters

ERROR: for app  Cannot create container for service app: create .: volume name is too short, names should be at least two alphanumeric characters
ERROR: Encountered errors while bringing up the project.```

Reply:
What's your docker and docker compose version?

Reply:
``` bash
[powilliam@dev ~]$ docker -v
Docker version 19.03.8-ce, build afacb8b7f0
[powilliam@dev ~]$ docker-compose -v
docker-compose version 1.25.5, build unknown
```

Reply:
calling `echo $(pwd)` in your current directory, what's ouput?

Reply:
It's the relative path to the folder. The problem is that docker and docker-compose isn't configured correctly. I fixed the problem. Sorry about that, i'll approve right know.